Task.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+
+class Task
+{
+    public $current_status = "";                  //Текущий статус, обязательное
+    public $action = "";                          //Пришедшее действие
+    public $customer_id = 0;                      //ID заказчика, обязательное
+    public $executor_id = 0;                      //ID исполнителя, обязательноe
+    public $role = "";                            //Роль пользователя, совершившего действий
+
+    const STATUSES = [
+        'new',                    //Статус новое
+        'canceled',               //Статус отменено
+        'in_progress',            //Статус в работе
+        'done',                   //Статус выполнено
+        'failed'                  //Статус провалено
+    ];
+
+    const ACTIONS = [
+       'to_cancel'       => ['cancel'],             //Отменить заказ
+       'to_take_to_work' => ['take_to_work'],       //Откликнуться на заказ
+       'to_refuse'       => ['refuse'],             //Отказаться от выполнения заказа
+       'to_confirm'      => ['confirm']             //Подтвердить выполнение заказа
+    ];
+
+    public function __construct($customer_id, $executor_id, $current_status, $role) {
+        $this->customer_id = $customer_id;
+        $this->executor_id = $executor_id;
+        $this->current_status = $current_status;
+        $this->role = $role;
+    }
+
+// Определять список из всех доступных действий
```

Comment:
Комментарии к методам нужно оформлять как phpDoc
```suggestion
    /**
     * Определять список из всех доступных действий
     */
```